app/services/exports/trainee_search_data.rb

```diff
@@ -52,7 +52,7 @@ def award_state(trainee)
       {
         recommended_for_award: I18n.t("exports.trainees.award_recommended", award_type: trainee.award_type),
         awarded: I18n.t("exports.trainees.award_given", award_type: trainee.award_type),
```

Comment:
Just wondering if there needs to be consistency across the application, e.g on the home page one of the cards is 'QTS awarded' whereas here it has been translated to 'Awarded QTS'.

Reply:
I've pushed another commit which makes things consistant. Let me know what you think.

Reply:
looks good to me!